pkg/cmd/cli/cli.go

```diff
@@ -69,6 +69,8 @@ func NewCommandCLI(name string) *cobra.Command {
 	cmds.AddCommand(cmd.NewCmdStartBuild(f, out))
 	cmds.AddCommand(cmd.NewCmdCancelBuild(f, out))
 
+	cmds.AddCommand(cmd.NewCmdRollback(name, "rollback", f, out))
```

Comment:
Only rollbacks, was arguing for clarity and standardization. 
On a second though though `rollback` would be compliant with other action-specific commands like `create`, `update`, `process`, etc. *And* the short description is good. So I'm ok with `rollback`, the only additional suggestion is to make the required argument part of the `Use` description (so it appears in the main help). Example: `Use: "rollback --from=<deployment>"`.